docusaurus/docs/learn/quickstarts/network/miniziti.bash

```diff
@@ -555,24 +657,28 @@ main(){
     #
 
     logDebug "fetching admin password from k8s secret to log in to ziti mgmt"
-    kubectl get secrets "ziti-controller-admin-secret" \
-        --namespace "${ZITI_NAMESPACE}" \
-        --output go-template='{{index .data "admin-password" | base64decode }}' \
-        | xargs ziti edge login "miniziti-controller.${MINIZITI_INGRESS_ZONE}:443" \
-            --yes --username "admin" \
+            getAdminSecret | xargs ziti edge login "miniziti-controller.${MINIZITI_INGRESS_ZONE}:443" \
```

Comment:
not `miniziti_login()`?

Reply:
No, because `miniziti_login()` presumes a pre-exisitng `ziti-cli.json`, and this does not.